docs/user-guide/analysis.md

```diff
@@ -59,15 +63,167 @@ be found
 
 #### Example: Reserving a single node for 20 minutes for interactive analysis
 
+    ```bash
     auser@uan01:> salloc --nodes=1 --tasks-per-node=1 --cpus-per-task=1 \
                   --time=00:20:00 --partition=standard --qos=short \
                   --reservation=shortqos --account=[budget code]
-
+    ```
 
 ## Data analysis nodes
 
-The full ARCHER2 system will have dedicated data analysis nodes. Further
-information on these nodes will be added here once it becomes available.
+The data analysis nodes on the ARCHER2 full system are designed for large
+compilations, post-calculation analysis and data manipulation. They should 
+be used for jobs which are too small to require a whole compute node, but 
+which would have an adverse impact on the operation of the login nodes if 
+they were run interactively.
+
+Unlike compute nodes, the data analysis nodes are able to access the `/home`, 
+`/work`, and `/rdfaas` directories. They can also be used to transfer data 
+from a remote system to ARCHER2 and vice versa (using e.g. `scp` or `rsync`).
+This can be useful when transferring large amounts of data that might take 
+hours to complete.
+
+The ARCHER2 data analysis nodes can be reached by using the `serial` 
+partition and the `serial` queue. Unlike other nodes on ARCHER2, you may 
+only request part of a single node and you will likely be sharing the node 
+with other users.
+
+The data analysis nodes are set up such that you can specify the number of 
+cores you want to use (up to 32 physical cores) and the amount of memory you 
+want for your job (up to 128 GB). You can have multiple jobs running on the 
+data analysis nodes at the same time, but the total number of cores used by 
+jobs currently running can not exceed 32 cores, and the total memory used 
+by jobs currently running cannot exceed 128 GB -- any jobs above this limit 
+will remain pending until your previous jobs are finished.
+
+!!! note:
```

Comment:
I think it would be useful to have a specific section titled something like "Requesting resources on data analysis nodes in Slurm" where we explicitly cover these details. We also need to mention how much memory you get by default if you just request cores.